util/dvsim.py

```diff
@@ -127,6 +134,17 @@ def main():
                       {test_name} set of directories where the test outputs go"""
     )
 
+    parser.add_argument(
+        "-pr",
+        "--proj-root",
+        metavar="path",
+        help="""root scratch directory path where all build / run drectories go;
+                      by default, the tool will create the {scratch_path} = {scratch_root}/{dut}
+                      directory if it doesn't already exist; under {scratch_path}, there will be
+                      {compile_set} set of directories where all the build outputs go and
+                      {test_name} set of directories where the test outputs go"""
```

Comment:
Update this description.
"Specify the root directory of the project. If this option is not passed, the tool will assume that this is a local GitHub repository and will attempt to automatically find it."

Reply:
Done